exo-go/src/app_config_helpers/index.go

```diff
@@ -50,3 +50,14 @@ func UpdateAppConfig(reader *bufio.Reader, serviceRole string, appConfig types.A
 		log.Fatalf("Failed to write application.yml: %s", err)
 	}
 }
+
+// GetSilencedDependencies returns a slice of silenced application dependenices
```

Comment:
Maybe mention in here that its the names of the dependencies versus some dependency object